flight/targets/aq32/board-info/board-info.mk

```diff
@@ -9,8 +9,8 @@ BOARD               := STM32F4xx_AQ32
 MODEL               := HD
 MODEL_SUFFIX        := 
 
-OPENOCD_JTAG_CONFIG := stlink-v2-norst.cfg
-OPENOCD_CONFIG      := stm32f4xx.stlink.cfg
+OPENOCD_JTAG_CONFIG ?= stlink-v2.cfg
```

Comment:
Good idea to make it possible to set this externally. I noticed that some of the newer nucleo boards need `stlink-v2-1.cfg`.